### DISCUSSION TITLE
How do I change the default meeting time on the schedules?

### DISCUSSION THREAD

**Discussion by ajtruckle** [ajtruckle/meeting-schedule-assistant-support#31] [Q&A] [answered]
I received this question:

> Morning, I can’t change the meeting time for the January schedule. What do I forget?

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/7985331/0fcac3dd-c324-43a6-a198-1e3d88cf18f9)

**Comment by ajtruckle** [comment, marked as answer]
The _Public Meeting_ time setting on the **Settings** window is now _obsolete_. The weekend [Meeting Time](https://www.publictalksoftware.co.uk/help-msa/msa-congdb-manage-congregations.html#MEETING_TIME) is now managed via the **Congregations** database.

Use the [Time Changes Manager](https://www.publictalksoftware.co.uk/help-msa/msa-congdb-times-manager.html) to configure the time to automatically change in January for your congregation:

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/7985331/851800c9-e98c-49e2-a230-f78452fe21e3)

**Reply by stecchio66** [reply]
Thanks for suggestion.

**Reply by stecchio66** [reply]
I noticed that the Italian translation of the "Override time" checkbox is wrong, so I didn't understand how to update the time in an existing schedule. 

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/118082203/a9294f3e-135e-4cbf-ae56-2ebec2f3a46c)

Clicking this checkbox overrides the default time set in this window.

**Reply by ajtruckle** [reply]
@stecchio66 Please provide the correct translation for Italian and I will update the software.

**Reply by stecchio66** [reply]
The translation is: "Sovrascrivere orario"
